src/main.js

```diff
@@ -105,7 +110,28 @@ function switchToViewSavedCovers(currentView) {
   savedView.classList.remove("hidden");
 }
 
-// We've provided two functions to get you started
+// MAKE MY BOOK BUTTON //
+createBookBtn.addEventListener("click", function(event){
+  event.preventDefault();
+  makeBookButton();
+});
+
+function makeBookButton() {
+  currentCover = createCover(userCover.value, userTitle.value, userDecr1.value, userDecr2.value);
+  coverImage.src = currentCover.coverImg; 
+  coverTitle.innerHTML = currentCover.title;
+  coverDescriptor1.innerHTML = currentCover.tagline1;
+  coverDescriptor2.innerHTML = currentCover.tagline2;
+  saveNewInfo();
+  goHome();
+}
+
+function saveNewInfo() {
+ covers.push(currentCover.coverImg);
+ titles.push(currentCover.title);
+ descriptors.push(currentCover.tagline1, currentCover.tagline2);
+}
```

Comment:
For clarity, I think i would pass in the updated `currentCover` as an argument for `saveNewInfo()`

Reply:
Since it's declared within the file and outside of the function, it's a global variable so it doesn't need to be passed as an argument.

Reply:
Yah totally true - I shouldn't have made this blocking!